### ISSUE TITLE
Errors loading mixin config in 1.0.6

### ISSUE THREAD

**Issue by jorgsaa** [IMB11/MRU#9]
Version: 1.0.6
MC 1.21.1
Neoforge 21.1.84

Looks like the mixin issues did not get resolved in 1.0.6
1.0.4 loads fine

```
Exception caught from launcher
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:67)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:105)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
Caused by: org.spongepowered.asm.launch.MixinInitialisationError: Error initialising mixin config mru.mixins.json
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.Config.create(Config.java:168)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.Mixins.createConfiguration(Mixins.java:121)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.Mixins.addConfiguration(Mixins.java:108)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.Mixins.addConfiguration(Mixins.java:98)
	at MC-BOOTSTRAP/fml_loader@4.0.34/net.neoforged.fml.loading.mixin.DeferredMixinConfigRegistration.lambda$registerConfigs$0(DeferredMixinConfigRegistration.java:51)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at MC-BOOTSTRAP/fml_loader@4.0.34/net.neoforged.fml.loading.mixin.DeferredMixinConfigRegistration.registerConfigs(DeferredMixinConfigRegistration.java:51)
	at MC-BOOTSTRAP/fml_loader@4.0.34/net.neoforged.fml.loading.mixin.FMLMixinPlatformAgent.prepare(FMLMixinPlatformAgent.java:24)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.launch.platform.MixinContainer.prepare(MixinContainer.java:122)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.launch.platform.MixinPlatformManager.prepare(MixinPlatformManager.java:166)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.launch.MixinBootstrap.doInit(MixinBootstrap.java:196)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.launch.MixinLaunchPluginLegacy.initializeLaunch(MixinLaunchPluginLegacy.java:200)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.launch.MixinLaunchPlugin.initializeLaunch(MixinLaunchPlugin.java:34)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.LaunchPluginHandler.lambda$announceLaunch$11(LaunchPluginHandler.java:106)
	at java.base/java.util.LinkedHashMap.forEach(LinkedHashMap.java:986)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.LaunchPluginHandler.announceLaunch(LaunchPluginHandler.java:106)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:51)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.Launcher.run(Launcher.java:103)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.Launcher.main(Launcher.java:74)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@2.0.2/cpw.mods.bootstraplauncher.BootstrapLauncher.run(BootstrapLauncher.java:210)
	at cpw.mods.bootstraplauncher@2.0.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:69)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	... 5 more
Caused by: java.lang.IllegalArgumentException: The specified resource 'mru.mixins.json' was invalid or could not be read
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinConfig.create(MixinConfig.java:1405)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.Config.create(Config.java:163)
	... 29 more
Caused by: java.lang.NullPointerException: Cannot invoke "org.spongepowered.asm.mixin.transformer.MixinConfig.onLoad(org.spongepowered.asm.service.IMixinService, String, org.spongepowered.asm.mixin.MixinEnvironment, org.spongepowered.asm.mixin.extensibility.IMixinConfigSource)" because "config" is null
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinConfig.create(MixinConfig.java:1398)
	... 30 more
Exiting with ERROR
```

**Comment by VaporeonScripts** [comment]
same error

**Comment by JohnGoten** [comment]
Can confirm as well https://mclo.gs/mloGVPP

EDIT: Issue only crept up since M.R.U 1.0.5+, luckily it is possible to run 1.0.4 with Sounds 2.3.0 https://mclo.gs/hg4LWS2

**Comment by IMB11** [comment]
I will look into this, apologies

**Comment by IMB11** [comment]
Should be fixed in 1.0.7
